src/Button/Button.js

```diff
@@ -0,0 +1,103 @@
+import React from 'react'
+import styled from 'styled-components'
+import PropTypes from 'prop-types'
+import { fontSize, space } from 'styled-system'
+import { theme } from '../'
+
+const size = props => {
+  switch(props.size) {
+    case 'small':
+      return { 
+        height: '32px',
```

Comment:
Fixed heights on buttons and inputs used to be sketchy with Firefox – make sure you do some monkey testing in a few browsers to make sure it looks okay

Reply:
Sure!